endoreg_db/views/video_segmentation_views.py

```
@@ -125,13 +127,13 @@ def stream(self, request, pk=None):
             raise Http404("Video streaming failed")
 
 
-# Neue separate View für Video-Streaming außerhalb des ViewSets
+@drf_permission_classes(PERMS)
 class VideoStreamView(APIView):
```

Pylint (3.3.7)

[refactor] Too few public methods (1/2)

(R0903)